backend/tests/fixtures.py

```diff
@@ -1,9 +1,23 @@
+import websockets
 import pytest
-from server import Server
-
+from ..server import Server
 
+WEBSOCKET_URI = "ws://localhost:8000"
 @pytest.fixture
 def server(event_loop):
 	server = Server({})
 	task = server.get_server_task(server.router)
 	event_loop.run_until_complete(task)
+
+@pytest.fixture
+async def client():
+	async with websockets.connect(WEBSOCKET_URI) as websocket:
+		yield websocket
+
+@pytest.fixture
+async def send_message(server, client):
```

Comment:
hm maybe this should be named something that makes it clear its going to get and return the reply?  I think I can imagine just a fire and forget message as well

Reply:
Hmm maybe with mypy we can just make it more explicit? If not then yeah it might be nice to make it more verbose

Reply:
i think it might be difficult to indicate that with mypy, this is an awfully dynamic magical thing.  BUT Ill see what I can come up with

A better name would probably be good anyway.